README.md

```diff
@@ -32,19 +32,19 @@ which has fundamental issues (like no comments).
 
 ## This solution
 
-Put all of your scripts in a file called `package-scripts.js` and use `p-s` in a single `package.json` script:
+Put all of your scripts in a file called `npsfile.js` and use `nps` in a single `package.json` script:
 
 **package.json**
 
 ```json
 {
   "scripts": {
-    "start": "package-scripts"
+    "start": "npm-package-scripts"
   }
 }
 ```
 
-**package-scripts.js**
+**npsfile.js**
```

Comment:
Good point :+1:

What if we simplify things and just have it be: `package-scripts.config.js`

That would make life MUCH easier and we wouldn't need to rename the project either :)